packages/frontend/src/components/InputCreator/index.tsx

```diff
@@ -60,6 +62,19 @@ export default function InputCreator(props: InputCreatorProps): JSX.Element {
     return <></>
   }
 
+  if (type === 'boolean-radio') {
+    return (
+      <BooleanRadio
```

Comment:
comment: if you didn't want to create a new component, one other way to solve the problem of `SingleSelect` only taking in string values is to leverage JSON. 

Scrappy example (`ControlledAutocomplete.tsx`)
```
<SingleSelect
  name="choose-dropdown-option"
  // ... omitted props for brevity
  items={formComboboxOptions(
    sessionOptions.map((option) => ({
      ...option,
      value: JSON.stringify(option.value),
    })),
    showOptionValue,
  )}
  onChange={(selectedOption) =>
    onChange(JSON.parse(selectedOption))
  }
  value={getSingleSelectOption(
    sessionOptions, // Need to JSON.stringify value here too
    field.value,
    freeSolo,
  )}
  // ... omitted props for brevity
```

Reply:
will be keeping the `boolean-radio` for UI and better coding semantics as discussed, thanks!